lib/msf/util/exe.rb

```diff
@@ -1437,6 +1437,18 @@ def self.to_powershell_command(framework, arch, code)
                     method: 'reflection')
   end
 
+  def self.to_ducky_script(framework, arch, code)
```

Comment:
I'm thinking this should be renamed `to_powershell_ducky_script`. Making it more specific like that will ensure that in the future we can support the ducky-script format for additional platforms like Linux and OSX. I know it's mostly used with Windows, but in theory it could work on anything since it just emulates a HID right? Let's leave that door open for us in the future.